fabtests/benchmarks/benchmark_shared.c

```diff
@@ -158,19 +158,21 @@ int pingpong_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 	if (ft_check_opts(FT_OPT_VERIFY_DATA))
 		inject_size = 0;
 
-	ret = ft_sync();
-	if (ret)
-		return ret;
-
```

Comment:
I missed the context of this change. Why do we need such commit? @wenduwan

Reply:
Are you saying that fi_write can happen before that overwrite is done on receiver side, if we don't make the overwrite before sync?

Reply:
Yes. For example, consider client(sender) and server(receiver):
- Receiver posts receive(RX1)
- Sender sends msg with delivery complete(TX1)
- Receiver gets receive completion(RX1)
- Receiver sends reply with delivery complete(TX2)
- Sender gets send completion(TX1)
- Sender posts receive(RX2)
- Sender gets receive completion(RX2)
- Sender starts fi_write
- Receive RX buffer modified by fi_write
- Receiver gets send completion(TX2)
- Receiver modifies RX buffer